OCast/Core/Device/DeviceDiscovery.swift

```diff
@@ -111,6 +111,7 @@ import CocoaAsyncSocket
     private var mSearchTimeout: TimeInterval
     private var mSearchIdx: Int
     private var mSearchRetry: Int
+    private let regularExpression = try? NSRegularExpression(pattern: "^uuid:([^:]*)")
```

Comment:
if "regularExpression" is not setted, can DeviceDiscovery work properly ?

Reply:
yes but it's not possible unless the regular expression is updated with a bad pattern. However I will add a fallback if the UDN format is not correct.